src/Agent.Worker/TestResults/JunitResultReader.cs

```diff
@@ -84,10 +101,21 @@ public TestRunData ReadResults(IExecutionContext executionContext, string filePa
             }
             else
             {
+                executionContext.Warning("Only single test suite found, parsing its information");
```

Comment:
This shouldn't be warning. It should be verbose output. use executionContext.Output